src/sections/components/componentCards.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RenderIf from 'react-rainbow-components/components/RenderIf';
+import PrismicComponentCards from '../../components/PrismicComponentCards';
+
+function filter(query, data) {
+    return data.filter((item) => {
+        const regex = new RegExp(query, 'i');
+        return regex.test(item.data['component-name'][0].text);
+    });
+}
+
+const sectionsName = [
+    { name: 'Layout Components' },
+    { name: 'Forms' },
+    { name: 'Miscellaneous' },
+    { name: 'Data View' },
+];
+
+function SectionTitle({ tag, componentsFiltered }) {
+    return sectionsName.map((component, index) => {
+        const { name } = component;
+        const key = `title-${index}`;
+        return (
+            <RenderIf key={key} isTrue={!!componentsFiltered.length && tag === name}>
```

Comment:
you don't need this iteration
you can put directly the RenderIf where SectionTitle and check only for:
```!!componentsFiltered.length```
also use sectionName or category instead of tag